Makefile

```diff
@@ -94,10 +101,6 @@ release:
 
 .PHONY: test
 test:
-	$(GO) fmt ./...
```

Comment:
These three cmd is no longer needed ?

Reply:
because the docker-compose can not run in docker,
so remove it at the moment,
we should refactoring the Makefile later